File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529-maximum-count-of-positive-integer-and-negative-integer.java
class Solution {
    public int maximumCount(int[] nums) {
// inintialize pos and neg
        int pos = 0;
        int neg = 0;
// if num > 0 increment pos, num < 0 increment neg
        for (int i=0; i<nums.length; i++){
            if (nums[i] < 0) neg++;
            else if (nums[i] > 0) pos++;
        }
// return the max of both
        if (pos >= neg) return pos;
        else return neg;
    }
}

File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529-maximum-count-of-positive-integer-and-negative-integer.py
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
# pos, neg to store the number of pos and neg numbers
        pos = 0
        neg = 0
# iterate through each element in nums if +ve increment pos, if -ve increment neg
        for num in nums:
            if num > 0:
                pos += 1
            elif num < 0:
                neg += 1
# return the maximum among pos and neg count
        return max(pos, neg)